schema/index.js

```diff
@@ -101,7 +102,8 @@ const schema = new GraphQLSchema({
     fields: {
       followArtist: FollowArtist,
       updateCollectorProfile: UpdateCollectorProfile,
-      updateConversation: UpdateConversation,
+      updateBuyerOutcome: UpdateBuyerOutcome,
+      appendConversationThread: AppendConversationThread,
```

Comment:
Hmm... we already have another conversation mutation for updating the buyer outcome, and that one was called `updateConversation`.